src/storage/index/LookupBaseProcessor.inl

```diff
@@ -123,6 +92,14 @@ bool LookupBaseProcessor<REQ, RESP>::isOutsideIndex(Expression* filter,
             }
             break;
         }
+        case Expression::Kind::kEdgeSrc:
+        case Expression::Kind::kEdgeType:
+        case Expression::Kind::kEdgeRank:
+        case Expression::Kind::kEdgeDst: {
+            auto* sExpr = static_cast<PropertyExpression*>(filter);
+            auto propName = *(sExpr->prop());
+            return propsInEdgeKey.find(propName) == propsInEdgeKey.end();
+        }
```

Comment:
What's the purpose here? I think here need  log `FATAL` or `ERROR`.

Reply:
These expression could convert to `PropertyExpression`, we could support filter like `prefix(dst) == "abc"` later.